vrk-kpa/suomidigi
# SUOPA-909 Tags

To test: 
- `make fresh`
- Log in and **Clear the caches from the UI**
- Go to f.e. https://suomidigi.docker.sh/node/500 (should be article) and scroll down to author information and check that there are tags and that you can click them (see image below)
![image](https://user-images.githubusercontent.com/1712902/76229236-8b185e80-622a-11ea-8dc6-1d528268ae04.png)
- Go to f.e https://suomidigi.docker.sh/node/775 (should be blog post) and scroll down to author information and check that there are tags and that you can click them
